src/app/series/directives/series-feed.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { DistributionModel, SeriesModel, StoryModel, TabService } from '../../shared';
+
+@Component({
+  template: `
+  <div>
+    <publish-spinner *ngIf="!isLoaded"></publish-spinner>
+    <section *ngIf="series && list">
+      <div class="hint" *ngIf="noStories">
+        You have no published stories in this series.
+      </div>
+
+      <section *ngIf="!noStories">
+        <ul *ngIf="privateStories.length">
+          <h4>Private</h4>
+          <li *ngFor="let s of privateStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="private">No publication date set.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="futurePublicStories.length">
+          <h4>Set for future publication</h4>
+          <li *ngFor="let s of futurePublicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="futurePublic">Will be published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+
+        <ul *ngIf="publicStories.length">
+          <h4>Public</h4>
+          <li *ngFor="let s of publicStories">
+            <h5><a [routerLink]="['/story', s.id]">{{s.title}}</a></h5>
+            <p class="public">Published {{s.pubDate}}.</p>
+          </li>
+        </ul>
+
+        <div class="extra" *ngIf="isLoaded">
+          <a [routerLink]="['/search', { tab: 'stories', seriesId: series.id }]">
+            Search among these stories.
+          </a>
+        </div>
+      </section>
+    </section>
+    </div>
+  `,
+  styleUrls: ['./series-feed.component.css']
+})
+
+export class SeriesFeedComponent implements OnDestroy {
+
+  isLoaded: boolean = false;
+  noStories: boolean;
+  series: SeriesModel;
+  publicStories: StoryModel[] = [];
+  futurePublicStories: StoryModel[] = [];
+  privateStories: StoryModel[] = [];
+  list: DistributionModel;
+  tabSub: Subscription;
+
+  constructor(tab: TabService) {
+    this.noStories = false;
+    this.tabSub = tab.model.subscribe((s: SeriesModel) => {
+      this.series = s;
+      s.loadRelated('distributions').subscribe((dists) => {
+        this.list = dists[0];
```

Comment:
Seems like this `list` (DistributionModel) isn't actually used for anything.  And as much as I like to see `loadRelated` being used ... you can probably delete this whole block and just do `sortStories()` after setting `this.series = s`.